interchaintest/setup/setup_celestia.go

```diff
@@ -21,9 +21,10 @@ var (
 	celestiaCoinDecimals = int64(6)
 	celestiaChainID      = "celestia-1"
 	celestiaAppImage     = ibc.DockerImage{
-		Repository: "ghcr.io/strangelove-ventures/heighliner/celestia",
-		Version:    "v1.8.0",
-		UidGid:     "1025:1025",
+		Repository: "celestia",
```

Comment:
good to switch back to ghcr?

Reply:
Not yet, looks like the runner may not be back up? Latest version built is still v1.9.0.